website/build.gradle

```diff
@@ -152,36 +188,49 @@ def getBaseUrl() {
   return "${System.getProperty('user.name')}-${buildContext}"
 }
 
+def getBranchRepo() {
```

Comment:
Could you add a comment on what is this supposed to do?

Reply:
Sure, I just added a description for this function